requirements.txt

```
@@ -7,7 +7,7 @@ torch>=2.3.0a0
 transformers>=4.41.2
 datasets>=2.15.0
 numba
-numpy==1.26.4
+numpy>=1.26.4,<2.0.0
```

Please change this line to:
```suggestion
numpy<2
```

I believe the `1.26.4` was arbitrarily pinned, it just needs to be under 2.

---

got it.  I'm actually going to copy over what the CLI repo has, because it had some more specific reasons on versions